src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -48,7 +49,7 @@ storiesOf('JsonSchemaViewer', module)
         <>
           <SchemaRow treeListNode={node} rowOptions={rowOptions} />
           <div className="flex h-full items-center">
-            <Button className="pl-1 mr-1" small minimal icon={<Icon color="grey" iconSize={12} icon="issue" />} />
+            <Button className="pl-1 mr-1" size="sm" appearance="minimal" icon="issue" />
```

Comment:
Should almost never have to use `className` - prefer `pl` and `mr` props instead please.